packages/dds/merge-tree/src/index.ts

```diff
@@ -39,3 +39,4 @@ export * from "./segmentPropertiesManager";
 export * from "./sortedSegmentSet";
 export * from "./textSegment";
 export * from "./referencePositions";
+export * from "./revertibles";
```

Comment:
there is active investigation on how to manage exports, but i don't think we have concrete guidance yet.

I explicitly wrote this code such that it could be easily tree shaken if not used. This was done by not coupling this code directly with the existing code (no references from other code here).

Reply:
went ahead and made it explicit, since i had other changes to make